<?xml version="1.0"?>
<launch>
    <rosparam file="$(find eurobot_localization)/config/secondary_robot_localization_params.yaml" command="load" ns="secondary_robot"/>
    <group ns = "secondary_robot">
        <node pkg="eurobot_localization" name="particle_filter_node" type="particle_filter_node2.py" respawn="true" output="screen"/>
    </group>
</launch>